lib/u3d/installer.rb

```diff
@@ -327,20 +337,26 @@ def install_sh(file, installation_path: nil)
   end
 
   class WindowsInstaller
-    def sanitize_install(unity)
+    def sanitize_install(unity, dry_run: false)
       source_path = File.expand_path(unity.path)
       parent = File.expand_path('..', source_path)
       new_path = File.join(parent, UNITY_DIR % unity.version)
-      UI.important "Moving #{source_path} to #{new_path}..."
+
+      UI.important "Moving #{source_path} to #{new_path}..." unless dry_run
```

Comment:
@niezbop this log is displayed before the conversion to windows paths (as opposed to what we do on oher platforms). Maybe we can move it back into the `if`. I can't really test on windows. Maybe you can?